app/controllers/sessions_controller.rb

```diff
@@ -4,14 +4,22 @@ def googleAuth
         # Get access tokens from the google server
         access_token = request.env["omniauth.auth"]
         user = User.from_omniauth(access_token)
-        log_in(user)
         # Access_token is used to authenticate request made from the rails application to the google server
         user.google_token = access_token.credentials.token
         # Refresh_token to request new access_token
         # Note: Refresh_token is only sent once during the first request
         refresh_token = access_token.credentials.refresh_token
         user.google_refresh_token = refresh_token if refresh_token.present?
```

Comment:
Do we really need these lines?

Reply:
I think they allow google to request new logins for security purposes (the google auth expires) -- maybe @michellekroll knows more?

Reply:
I am not sure but I can look into it. I believe the functionality we have right now will not do that though.